packages/builders/src/messages/Message.ts

```diff
@@ -197,6 +212,15 @@ export class MessageBuilder implements JSONEncodable<RESTPostAPIChannelMessageJS
 		return this;
 	}
 
+	/**
+	 * Sets the embeds for this message.
+	 *
+	 * @param embeds - The embeds to set
+	 */
+	public setEmbeds(...embeds: RestOrArray<APIEmbed | EmbedBuilder | ((builder: EmbedBuilder) => EmbedBuilder)>): this {
+		return this.spliceEmbeds(0, this.embeds.length, ...normalizeArray(embeds));
```

Comment:
I tested it and while it works, I would much rather stick to explicitly overriding the array instead on relying on splicing to replicate a set operator

Reply:
This (and the other changes like this) fall under the "Consistency:tm:" part of my description

Reply:
Well I am against relying on splice to do a set 🤷, would love to hear other's thoughts.

Reply:
I don't see a problem with using the splice method, but if we're not going to, then we should update it everywhere else too.

Reply:
I think this can be decided and updated in another pull request. Doesn't seem like a good reason to block merging this